include/deal.II/lac/cuda_kernels.templates.h

```diff
@@ -573,9 +591,9 @@ namespace LinearAlgebra
 
       template <typename Number>
       __global__ void
-      add_permutated(Number *         val,
+      add_permutated(const size_type *indices,
```

Comment:
Why doesn't have this function an `IndexType` template parameter? Would it make sense for conformity?

Reply:
Because it doesn't need to be templated. I only template the functions that have to be templated

Reply:
OK